packages/freezed/lib/src/utils.dart

```diff
@@ -71,3 +71,16 @@ String _fixCase(String input, String separator) =>
 
       return lower;
     });
+
+String typeStringWithoutNullability(String typeString) {
```

Comment:
Why do we need those utilities? `getDisplayString` has a `withNullability` parameter

Reply:
That is true but for not generated types (e.g. other Freezed classes) the common type checking is done on Strings not DartTypes.